GameData/RealismOverhaul/RO_SuggestedMods/NearFutureTechnologies/RO_NearFuture_Electrical_Nuclear.cfg

```diff
@@ -63,15 +81,24 @@
 	{
 		@type = RealismOverhaulStackHollow
 	}
-	@RESOURCE[DepletedUranium]
-	{
-		@maxAmount = 100
-	}
+	ù
```

Comment:
huh?

Reply:
that's the key right next to the enter key in the italian keyboard layout. I fat-finger a lot